common/test/acceptance/tests/discussion/test_discussion.py

```diff
@@ -153,6 +153,69 @@ def setup_thread_page(self, thread_id):
         self.thread_page = self.create_single_thread_page(thread_id)  # pylint: disable=attribute-defined-outside-init
         self.thread_page.visit()
 
+    def test_mathjax_rendering(self):
+        thread_id = "test_thread_{}".format(uuid4().hex)
+        mathjax_body = """I wrote up my solution to Problem 7.57 in the text in the hope that it is helpful. If you
```

Comment:
Can you make this a constant variable and put it outside the test class somewhere? Also the first example here has much nicer indentation: http://stackoverflow.com/a/10660443.